COM-71 Add new essential drugs

lauravignoli opened the pull request:
Retired existing essential drugs and added new ones.

Used a csv file instead of a sql script to retire the old drugs so that we don't have issues with synching openERP in future.

lauravignoli committed:
COM-71 Add new essential drugs

Retired existing essential drugs and added new ones.

Used a csv file instead of a sql script to retire the old drugs so that we don't have issues with synching openERP in future.

tmvumbi2 commented on configuration/drugs/1020_essential_drugs.csv:
```diff
@@ -0,0 +1,835 @@
+65758339-1c9b-4fff-adcc-b6364fcfb563,0,Diethylcabamazine Citrate 50mg sous blister,Diethylcabamazine Citrate,Comprimé,50mg,,
+cbb0d644-e2f7-440f-91c9-b4b3b10e18a2,0,Diéthylcarbamazine 100mg,Diethylcarbamazine,Comprimé,100mg,,
+5ad558ee-f4b7-4310-a212-4f5639f801bd,0,"Diféuines (I, II et III) 23mg+32mg+54mg",Diféuines,Sirop,23mg+32mg+54mg,,
+3fb25c98-6475-4060-b4d3-d791c493bff7,0,"Digoxine 0,25mg",Digoxin,Comprimé,"0,25mg",,
```
@lauravignoli small typo on "Digoxine" (missing e). Please see line 285 to 289 as well

tmvumbi2 commented on configuration/drugs/1020_essential_drugs.csv:
```diff
@@ -0,0 +1,835 @@
+43aa9903-8f5e-4af8-8c91-1009354b902c,0,Gentamicine 20mg/ml,Gentamicin,Injection,20mg/ml,,
+9898e33a-610b-4302-8f5b-2f310717f4b3,0,Gentamicine 40mg/ml,Gentamicin,Injection,40mg/ml,,
+9bda1035-a265-4915-814d-91ec6ca25488,0,Gentamicine 80mg/ml,Gentamicin,Injection,80mg/ml,,
+e9c8021f-5796-440b-803e-0559124c9b9c,0,"Gentamycine 0,3%",Gentamicin,Collyre,"0,30%",,
```
@lauravignoli Small typo on "Gentamycine" (missing "e" at the end); please see line 387 to 390 as well

tmvumbi2 commented on configuration/drugs/1020_essential_drugs.csv:
```diff
@@ -0,0 +1,835 @@
+f7356169-c455-4c7b-a8ba-5be6dc1ca844,0,Gamma (difeuines) FL/250ml,Gamma Syrup (Difeuines),Sirop,FL/250ml,,
+ff343e59-1f62-4965-be6e-9c37151154d8,0,Gatifloxacine 400mg,Gatifloxacin,Comprimé,400mg,,
+bf26ca37-c684-40bf-a375-1f00decfe33b,0,Gélatine fluide modifiée 500ml,Modified Fluid Gelatin,Injection,500ml,,
+00de8bbd-d69e-474c-b331-e6de3360fc40,0,Gentamicine 10mg/ml,Gentamicin,Injection,10mg/ml,,
```
@lauravignoli Not sure if this is a typo or not; should "Gentamicine" read  "Gentamycine"? please see line 383 to 385 as well. Note: this drug is also spelled Gentamicine in the excel sheet

lauravignoli committed:
COM-71 Use sql to retire drugs

The csv was not being processed correctly and is not retiring the drugs, so a sql script will be used instead.

lauravignoli committed:
COM-71 Fix drug name typos

tmvumbi2 committed:
COM-71 Remove duplicates from retire_unused_drugs.sql
Removes 5 uuids that were duplicated
